DAS/README.md

```diff
@@ -2,29 +2,122 @@
 
 *This directory contains scripts to log and test various components of the data data-acquisition-system*
 
-## **Basic Setup and Usage**
-These steps assume that you are in the data-acquisition-system root directory.
+
+
+## Basic Setup and Usage
+These steps assume that you are in the data-acquisition-system root directory and poetry installed.
 1. `cd DAS`
 2. `poetry install`
 3. `poetry shell`
-4. Run any DAS command, eg. `python -m das.mqtt_wireless_logger` (or `python mqtt_wireless_logger.py` as appropriate for your current directory)
+4. Run any DAS command, _e.g._ `python -m das.V3_mqtt_recorder`
+
+## Run Tests
+These steps assume that you have done the basic setup.
+1. `pytest`
+
+<br/>
+
+## [V3 MQTT Recorder](/DAS/das/V3_mqtt_recorder.py)
+The command line tool allows for mqtt messages on any topic to be logged and recorded to a csv file. The saved csv files will be stored in the `das/csv_data`. 
+
+### Usage
+```
+# General command
+python -m das.V3_mqtt_recorder [TOPICS] [FLAGS]
+
+# Subscribe to all topics on the V3 bike
+python -m das.V3_mqtt_recorder /v3/#  -v
+
+# Subscribe just BOOST and wireless module topics
+python -m das.V3_mqtt_recorder /v3/wireless-module/# power-model/# -v
```

Comment:
The topics have been updated in #66, so just confirming whether we'll be changing these to the updated topics in a separate PR or shall we do it here (if it's a separate PR, we should prioritise that so we don't have inconsistent documentations lying around)

Reply:
Since #66 has been merged, this PR should have its topics updated. No big deal, a very quick fix!